Dockerfile

```diff
@@ -12,24 +12,26 @@
 # the License.
 #
 
-FROM adoptopenjdk/openjdk11:alpine
+FROM eclipse-temurin:11-jdk-alpine as builder
+WORKDIR /build
+RUN apk add --no-cache git
+COPY ./ ./
+RUN ./mvnw package -DskipTests
 
-EXPOSE 8080
-EXPOSE 8081
-EXPOSE 8443
 
-RUN addgroup -S thirdeye && \
-  adduser -S thirdeye -G thirdeye && \
-  chown thirdeye:thirdeye /home/thirdeye
+FROM eclipse-temurin:11-jre-alpine
+RUN addgroup -g 1000 thirdeye && \
+  adduser -u 1000 thirdeye -G thirdeye -D -H
 
 USER thirdeye
-WORKDIR /home/thirdeye
+WORKDIR /app
 
-COPY thirdeye-distribution/target/thirdeye-distribution-*-dist.tar.gz thirdeye-distribution-dist.tar.gz
+EXPOSE 8080
+EXPOSE 8081
+EXPOSE 8443
 
-RUN mkdir thirdeye && tar -xvf thirdeye-distribution-dist.tar.gz -C thirdeye --strip-components=1 && \
-  rm thirdeye-distribution-dist.tar.gz
+ENV THIRDEYE_PLUGINS_DIR "/app/plugins"
```

Comment:
this should not be necessary. 
any reason for adding this?

Reply:
nice, removed